src/docs/sdk/sessions.mdx

```diff
@@ -145,6 +145,77 @@ The following fields exist:
 >   agent of the user that caused the session. This data is not persisted but
 >   used for filtering.
 
+## Session Aggregates Payload
+
+<Alert title="Experimental Status" level="info">
+
+The `"sessions"` envelope item is still unfinished and considered experimental.
+It is currently only supported on the `sentry.io` installation and is subject
+to heavy usage-limits.
+
+Currently, the sum of all session counts must not exceed _100_. This limitation
+will be lifted in the future when this feature will become stable.
+
+</Alert>
+
+Especially for _request-mode_ sessions (see below), it is common to have
+thousands of requests, and thus sessions, per second.
+
+Under the assumption that these sessions will be short-lived, and tracking their
+duration is not desired, these sessions can be aggregated together on the
+SDK side, before they are sent to Sentry.
+
+An SDK should aggregate closed sessions and group them by their `started` time
+and `distinct_id`. These groups will be sent as `sessions` envelope item.
+It consists a JSON payload that looks roughly like this:
+
+```json
+{
+  "aggregates": [
+    {
+      "started": "2020-02-07T14:16:00Z",
+      "exited": 123
+    },
+    {
+      "started": "2020-02-07T14:16:00Z",
+      "did": "optional distinct user id",
+      "exited": 12,
+      "errored": 3
+    }
+  ],
+  "attrs": {
+    "release": "release name",
+    "environment": "environment name",
```

Comment:
Since this is an example and you explain the payload below again, let's give actual example values here.